mmseg/ops/inverted_residual_module.py

```diff
@@ -0,0 +1,73 @@
+from mmcv.cnn import ConvModule, build_norm_layer
+from torch import nn
+
+
+class InvertedResidual(nn.Module):
```

Comment:
This class is not an op but a common block. It can be moved to `models.utils`.

Reply:
"Inverted Residual" has been moved to models.utils.